features/enabled_error_types.feature

```diff
@@ -1,144 +1,52 @@
 Feature: Enabled error types
 
-Background:
-    Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-
 Scenario: All Crash reporting is disabled
     # Sessions: on, unhandled crashes: off
-    When I crash the app using "DisableAllExceptManualExceptionsAndCrashScenario"
-    And I relaunch the app
-    # Sessions on, unhandled on
-    And I crash the app using "NullPointerScenario"
-    And I relaunch the app
-    And I wait for 3 requests
-    And the request 0 is valid for the session tracking API
-    And the request 1 is valid for the session tracking API
-    And the request 2 is valid for the error reporting API
-    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
-    # We only see the Null pointer exception
-    And the payload field "events.0.exceptions.0.errorClass" equals "EXC_BAD_ACCESS" for request 2
+    When I run "DisableAllExceptManualExceptionsAndCrashScenario" and relaunch the app
```

Comment:
Feels like we should have stronger expectations than just receiving any two session requests.  Can't we check the contents of each?